openapi/googleapis.vision/Ballerina.toml

```diff
@@ -1,12 +1,12 @@
 [package]
 license = ["Apache-2.0"]
-keywords = ["Business Intelligence/Artificial Intelligence", "Cost/Freemium", "Vendor/Google"]
+keywords = ["AI/Google", "Cost/Freemium", "Vendor/Google"]
 org = "ballerinax"
 name = "googleapis.vision"
 icon = "icon.png"
 distribution = "2201.2.1"
 repository = "https://github.com/ballerina-platform/ballerinax-openapi-connectors"
-version = "1.4.1"
+version = "1.4.2"
```

Comment:
If we have to release this connector with Bal 2201.3.x, we need to bump the connector version to 1.5.0

Reply:
Addressed in https://github.com/ballerina-platform/openapi-connectors/pull/753/commits/c82ce8da6f9d6bbd56a1ed56d8eb103b57261493

Reply:
Shall we update the ballerina distribution version too for 2201.3.2

Reply:
As choreo uses `2201.3.1`, we also use the same version.

@AathmanT Shall we update the bal version in Ballerina.toml and Package.md of this connector.

Reply:
Updated them